web/src/main/webapp/WEB-INF/jsp/team/match/detail.jsp

```diff
@@ -0,0 +1,218 @@
+<%@ page contentType="text/html;charset=UTF-8" pageEncoding="utf-8" trimDirectiveWhitespaces="true" %>
+<%@ taglib tagdir="/WEB-INF/tags" prefix="my" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
+<%@ taglib prefix="s" uri="http://www.springframework.org/tags" %>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form" %>
+<%@ taglib prefix="sec" uri="http://www.springframework.org/security/tags" %>
+
+<my:pagetemplate title="MatchDetail">
+    <jsp:attribute name="body">
+
+        <div class="container">
+            <div class="row">
+                <div class="col-md-8 offset-md-2">
+
+                    <legend class="m-t-3 text-md-center" id="match-header">
+                        <strong><c:out value="${matchResult.match.homeTeam.name}"/></strong>
+                        <span> </span>
+                        <c:out value="${matchResult.homeGoals} - ${matchResult.awayGoals}"/>
+                        <span> </span>
+                        <strong><c:out value="${matchResult.match.awayTeam.name}"/></strong>
+                    </legend>
+                    <p class="m-a-0 text-md-center">
+                        <strong><span>Start: </span></strong>
+                        <c:out value="${matchResult.match.startTime.format(formatter)}"/>
+                    </p>
+                    <p class="text-md-center">
+                        <strong><span>End: </span></strong>
+                        <c:out value="${matchResult.match.endTime == null ? ' - ' : matchResult.match.endTime.format(formatter)}"/>
+                    </p>
+                    <div class="container">
```

Comment:
tohle už by neměl být container, ne? imho je to zbytecne a stači ten vnitřní `row`